tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -1139,12 +1139,24 @@ static void VerifyPdbUtil(dxc::DxcDllSupport &dllSupport,
       VERIFY_SUCCEEDED(pPdbUtils->GetSourceName(i, &pFileName));
       VERIFY_SUCCEEDED(pPdbUtils->GetSource(i, &pFileContent));
       if (0 == wcscmp(pFileName, pMainFileName)) {
-        VERIFY_IS_TRUE(pFileContent->GetBufferSize() == MainSource.size());
-        VERIFY_IS_TRUE(0 == std::memcmp(pFileContent->GetBufferPointer(), MainSource.data(), MainSource.size()));
+        llvm::StringRef FileContentRef((const char *)pFileContent->GetBufferPointer(), pFileContent->GetBufferSize());
```

Comment:
We should probably just QI for IDxcBlobUtf8 (verify that succeeds) and then use GetStringPointer() and GetStringLength() here (which will exclude the null terminator).